ssr-playground/rollup-server.config.js

```diff
@@ -0,0 +1,62 @@
+// This config is used when running `server.js`.
+
+import lwc from '@lwc/rollup-plugin';
+import replace from '@rollup/plugin-replace';
+import alias from '@rollup/plugin-alias';
+import simpleRollupConfig from'./rollup.config.js';
```

Comment:
If we remove `csr-dev` then we can potentially move this into this file, and have one single `rollup.config.js` file.